go/protocol/keybase1/extras.go

```diff
@@ -754,13 +754,28 @@ func (b TLFIdentifyBehavior) AlwaysRunIdentify() bool {
 }
 
 func (b TLFIdentifyBehavior) WarningInsteadOfErrorOnBrokenTracks() bool {
+	// The chat GUI (in non-strict mode) is specifically exempted from broken
+	// track errors, because people need to be able to use it to ask each other
+	// about the fact that proofs are broken.
 	return b == TLFIdentifyBehavior_CHAT_GUI
 }
 
 // All of the chat modes want to prevent tracker popups.
 func (b TLFIdentifyBehavior) ShouldSuppressTrackerPopups() bool {
-	return b == TLFIdentifyBehavior_CHAT_GUI || b == TLFIdentifyBehavior_CHAT_CLI ||
-		b == TLFIdentifyBehavior_CHAT_GUI_STRICT
+	switch b {
+	case TLFIdentifyBehavior_CHAT_GUI,
+		TLFIdentifyBehavior_CHAT_GUI_STRICT,
+		TLFIdentifyBehavior_CHAT_CLI,
```

Comment:
I thought CLI chat still needs popups? Perhaps that changed. Just thought I'd point it out.